src/backend/joanie/edx_imports/management/commands/migrate_edx.py

```diff
@@ -125,6 +134,13 @@ def handle(self, *args, **options):
         batch_size = options["batch_size"]
         dry_run = options["dry_run"]
 
+        if check:
```

Comment:
I suggest to do the opposite. To check every time and use an option `skip-check` if you want to explicitly skip check. WDYT ?

Reply:
yes, good idea